src/Test.sol

```diff
@@ -159,7 +159,7 @@ abstract contract Test is DSTest {
         assertTrue(!data, err);
     }
 
-    function assertEq(bool a, bool b) internal {
+    function assertEq(bool a, bool b) internal virtual {
```

Comment:
Agree with keeping the `assertApproxEq` methods virtual in case users want different approximation logic. The `assertEq` probably don't need to be virtual, and existing ones in DSTest's `test.sol` are also not virtual, so we should probably follow that convention for now